ScreenSelectorExample/ScreenSelectorExampleProject/Assets/PostProcessLauncherCopy.cs

```diff
@@ -10,6 +10,12 @@ public class PostProcessLauncherCopy
     [PostProcessBuildAttribute(1)]
     public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProject)
     {
+        string dataPath = Path.Combine(Path.GetDirectoryName(pathToBuiltProject), "PersistentDataPath.txt");
+        using (StreamWriter file = new StreamWriter(dataPath))
+        {
+            file.WriteLine(Path.Combine(Application.persistentDataPath, "ScreenSelectorPrefs.txt"));
```

Comment:
That doesn't work. As soon as you move your Windows player to another machine, persistentDataPath will be different. This should probably write out application/publisher name and then compute the actual path at runtime.

Reply:
This is true but how common is this ? It wouldn’t be any different than today if I moved the game to another machine and it didn’t have any player prefs in the registry.

Reply:
Good catch. Will fix.